src/stickers.yml

```diff
@@ -28748,4 +28748,11 @@ ae1d343accdec586fe19954a6be7aee9:
   key: ffea9831b6af7617e4ab1b3868ce6f26f5c7c8ab3f05fb8d68b436f9253232ab
   tags:
     - cat
+    616d9d838816344f71ef06bd74d32b35:
```

Comment:
you're going to want to back this line out to zero leading spaces.

Reply:
Okay thanks, my bad.